api/models/condition.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+require('./trial');
+
+const bookshelf = require('../../config').bookshelf;
+const BaseModel = require('./base');
+
+const Condition = BaseModel.extend({
+  tableName: 'conditions',
+  visible: [
+    'id',
+    'name',
+    'type',
```

Comment:
`type` now is just `condition` or `other`, as far as I know. Maybe we should also remove this attribute? /cc @roll